zingolib/src/commands.rs

```diff
@@ -600,7 +600,11 @@ impl Command for ExportUfvkCommand {
         match ufvk_res {
             Ok(ufvk) => {
                 use zcash_address::unified::Encoding as _;
-                ufvk.encode(&lightclient.config().chain.to_zcash_address_network())
+                object! {
+                    "ufvk" => ufvk.encode(&lightclient.config().chain.to_zcash_address_network()),
+                    "birthday" => RT.block_on(lightclient.wallet.get_birthday())
```

Comment:
Why pack this into this API when there's a purpose specific `get_birthday` already?

Reply:
As with `seed`, the birthday is something you almost always want in the context where this function is called....and if you forget or don't know you need to export your birthday when you export your key/seed, it can be really inconvenient. So we return the birthday to make it obvious that it's something you probably want if you're asking for the seed/key.

Reply:
It also makes a more convenient interface for zingo-mobile/zingo-pc.